app/src/main/java/org/oppia/app/utility/FontScaleConfigurationUtil.kt

```diff
@@ -0,0 +1,35 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.Context.WINDOW_SERVICE
+import android.util.DisplayMetrics
+import android.view.WindowManager
+import org.oppia.app.model.StoryTextSize
+import javax.inject.Inject
+
+/** FontScaleConfigurationUtil helps to update font scale. */
```

Comment:
Or perhaps: ``Utility to change the scale of font for the entire app.``

Reply:
Done.